src/PKSim.Infrastructure/Services/DefaultIndividualRetriever.cs

```diff
@@ -39,7 +40,7 @@ public Individual DefaultHuman()
 
       public Individual DefaultIndividualFor(Species species)
       {
-         return DefaultIndividualFor(_populationRepository.DefaultPopulationFor(species));
+         return DefaultIndividualFor(species.DefaultPopulation);
```

Comment:
Using default population directly instead of populationRepo which uses selection from user